packages/css/test/instance/__snapshots__/css.test.js.snap

```diff
@@ -35,8 +35,6 @@ exports[`css array with explicit true 1`] = `
 exports[`css auto px 1`] = `
 .emotion-0 {
   display: flex;
-  -webkit-flex: 1;
```

Comment:
those disappeared because this file is testing a custom instance and I have not added a prefixer to it